noah/src/test/scala/edu/uci/ics/cloudberry/noah/kafka/TestKafka.scala

```diff
@@ -28,12 +28,6 @@ class TestKafka extends Specification with Mockito {
       argument.getValue.isInstanceOf[ProducerRecord[String, String]] must(beTrue)
     }
 
-    "close should be called" in {
```

Comment:
could you add one corresponding test when it should be closed, it closes successfully?